src/tactic/simp_rw.lean

```diff
@@ -0,0 +1,47 @@
+/-
+Copyright (c) 2020 Anne Baanen. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Anne Baanen
+
+The `simp_rw` tactic, a mix of `simp` and `rewrite`.
+-/
+import tactic.core
+
+/-!
+# The `simp_rw` tactic
+
+This module defines a tactic `simp_rw` which functions as a mix of `simp` and
+`rw`. Like `rw`, it applies each rewrite rule in the given order, but like
+`simp` it repeatedly applies these rules and also under binders like `∀ x, ...`,
+`∃ x, ...` and `λ x, ...`.
+
+## Implementation notes
+
+The tactic works by taking each rewrite rule in turn and applying `simp only` to
+it. It should be possible to support backwards rewriting in the tactic, i.e.
+`simp_rw [←lemma]`, but it will be more useful and not much more work to add
+support for this directly to `simp`.
+-/
+
+namespace tactic.interactive
+open interactive interactive.types tactic
+
+/-- `simp_rw` is a version of `simp` which performs rewriting in the given order.
```

Comment:
There docstring and the documentation in `docs/tactics.md` serve two purposes:
 * `docs/tactics.md` is the place for longer explanation and examples
 * the docstring should be more a concise reference description, such as listing the supported syntax for the arguments